platforms/nuttx/src/bootloader/common/image_toc.c

```diff
@@ -50,11 +50,16 @@
 #endif
 #define FLASH_END_ADDRESS (FLASH_START_ADDRESS + BOARD_FLASH_SIZE)
 
-bool find_toc(const image_toc_entry_t **toc_entries, uint8_t *len)
+/* If the ToC location is not explicitly stated, define a default value */
+#ifndef TOC_ADDRESS
+#define TOC_ADDRESS APP_LOAD_ADDRESS + BOOT_DELAY_ADDRESS + 8
```

Comment:
Maybe this is not necessary anymore, if we can break backwards compatibility with firmwares that have ToC + signature.

Reply:
This is probably not needed. Instead, define this as an example in px4/fmu-v5/src/hw_config.h . everyone can just define where their TOC lies